index.html

```diff
@@ -6,7 +6,7 @@
     <script
       defer
       class="remove"
-      src="https://www.w3.org/Tools/respec/respec-w3c">  
+      src="https://www.w3.org/Tools/respec/respec-w3c">
```

Comment:
little bit annoying... but maybe the white space changes could be a separate PR?